xilem/examples/mason.rs

```diff
@@ -32,9 +35,30 @@ fn app_logic(data: &mut AppData) -> impl WidgetView<AppData> {
         Axis::Vertical
     };
 
-    let sequence = (0..count)
-        .map(|x| button(format!("+{x}"), move |data: &mut AppData| data.count += x))
+    let flex_sequence = (0..count)
+        .map(|x| {
+            (
+                button(format!("+{x}"), move |data: &mut AppData| data.count += x),
+                if data.active {
+                    FlexSpacer::Flex(x as f64)
+                } else {
+                    FlexSpacer::Fixed((count - x) as f64)
+                },
+            )
+        })
         .collect::<Vec<_>>();
+
+    let flexy_flex_sequence = [2, 3, 4].map(|c| {
+        if data.count.abs() % c == 0 {
+            FlexSpacer::Fixed(30.0 * c as f64).into_any_flex()
+        } else {
+            button(format!("flexy +{c}"), move |data: &mut AppData| {
+                data.count += c;
+            })
+            .into_any_flex()
+        }
+    });
```

Comment:
This feels `FizzBuzz` adjacent - not sure that we should action that though

Reply:
Ah yeah, lets make it `FizzBuzz` ^^